workmanager/ios/workmanager/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "workmanager",
+    platforms: [
+        .iOS("13.0")
+    ],
+    products: [
+        .library(name: "workmanager", targets: ["workmanager"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/flutter/engine", from: "0.0.0")
```

Comment:
@ened IIRC `flutter/engine` has been archived and now lives under https://github.com/flutter/flutter/tree/master/engine

I'm unsure how this affects this setup.